broadpeakplugin/src/main/java/com/kaltura/playkit/plugins/broadpeak/BroadpeakPlugin.java

```diff
@@ -157,6 +158,10 @@ public void apply(PKMediaEntry mediaEntry, PKMediaEntryInterceptor.Listener list
             // Start the session and get the final stream URL
             session = SmartLib.getInstance().createStreamingSession();
             session.attachPlayer(player, messageBus);
+            if (session == null) {
```

Comment:
If session is null won't it also throw for calling attachPlayer?

Reply:
indeed
that i also not sure how exception happened